Update triggers-reference.md (#4150)
harness/developer-hub: docs/platform/triggers/triggers-reference.md

The personal access token used for connector authentication must have the appropriate scopes/permissions.

For example, a GitHub personal access token for a GitHub connector must include all `repo`, `user`, and `admin:repo_hook` options for **Scopes**.

> Also, please ensure the user to whom the PAT belongs should have atleast **Admin** permissions.

Reviewer: ```suggestion
Also, please ensure the user using the PAT has at least **Admin** permissions in the repo.
```
